admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -267,10 +267,14 @@ maintenance mode, and then disable your encryption module with this command::
 
  occ maintenance:mode --on
  occ encryption:disable
- 
+ occ encryption:decrypt-all
+
 Take it out of maintenance mode when you are finished::
 
  occ maintenance:mode --off
+ 
+.. warning:: Disabling encryption without decrypting all the files can lead to decryption errors in the futur as this state is not handled well by Nextcloud.
```

Comment:
s/futur/future/

maybe remove the part that it's not handled well but reword to say it can lead to unpredictable behavior ?

Reply:
Done.